### ISSUE TITLE
want to apply a style to the selected paragraphs of the page, not the entire page.

### ISSUE THREAD

**Issue by zhongguocn** [stevencohn/OneMore#851]
The current text style can only be applied to a paragraph or an entire page. 

I don't want to change the text format after I have spent time setting it. At this time, I only want to apply the style to the selected paragraphs of the page, not the entire page.

Thanks.

**Comment by stevencohn** [comment]
OneMore custom styles are controlled by the Style Type property of each style, where only the _Characters_ type is applied to the selection range within a paragraph and _Paragraph_ and _Heading_ are applied to the entire paragraph:

* Characters
* Paragraph
* Heading

<img width="343" alt="image" src="https://user-images.githubusercontent.com/9964757/215327371-1dba8ef1-274b-4dc0-8b9d-00e6520d256a.png">

If you want to apply a style to selected characters, either change the style definition's style type to _Characters_ or create a new style definition using the _Characters_ style type.

**Comment by stevencohn** [comment]
I think I misread the posting. Sorry, let me try again.

In addition to what I wrote above, you can easily apply a style to a selected paragraph. Just select that paragraph and then choose one of your custom styles from the OneMore Custom Styles menu.

**Comment by zhongguocn** [comment]
Thank you for your answer. I have been using the way you said to apply styles to a paragraph.
Expect to apply styles to multiple selected paragraphs.

Thank you for developing such a easy-to-use plug-in.

**Comment by julie777** [comment]
I think this issues suggests a terminology problem.  My understanding (without looking at any documentation) is that OneMore has:
- Page Theme
- Style Set
- Custom Style

Custom Style
A custom style is a specific set of formatting information for a word or paragraph, which is part of its definition.
Examples would be:
- Heading 1
- bold

Style Set
A named group of Custom Styles that includes the full set of Custom styles and their definitions and 
- allows changing the set in use by just changing the active Style Set in the menu
- allow customizing an entire set of styles so that they work together nicely. For example, in a given Style Set, Heading 1 is likely to use a larger font size than Heading 2

Page Theme
A named combination of page attributes, such as background color, background image, page lines, default container width, etc. in addition to the Style Set used for that page.

Changing

Changing a single style setting from the menu or with a shortcut such as `<ctrl><alt><shift>-1` will apply to the selection or the appropriate object containing the cursor. So if the cursor is in a word and I select "bold" that single word should change to "bold", but if the cursor is in a word, or at the end of a line and I select "Heading 1" then the paragraph containing the cursor should be formatted as "Heading 1"

Confusion occurs currently because "Style Set" is never actually used. Instead the menu has "Apply Styles to Page" which would be much clearer if it was Apply Style Set to Page, or even Apply Current Style Set To Page, if there is no dialog asking which style set to use. 

There is also a missing possibility which is "Apply <style> to Page" which would all choosing a single style such as bold and for every place in the page using bold switch it from its current bold to the bold in the currently set style. I will admit that this probably isn't necessary given that a Style Set is designed to work together and therefore to correct any style discrepancies just applying the Style Set to the Page would be sufficient.

**Comment by stevencohn** [comment]
Thanks @julie777. I think your comments here reflect what you also wrote in #746 which is currently under consideration - and some development.
